applications_tests/dem_3d/sliding_restart_files/sliding_restart.simulationcontrol

```diff
@@ -4,5 +4,5 @@ dt_1 1e-05
 dt_2 1e-05
 dt_3 1e-05
 CFL  0
-Time 0.15
-Iter 15000
+Time 0.25
+Iter 25000
```

Comment:
Why make this test longer? Won't this just make it more unstable?

Reply:
Yes, it could become more unstable. I will correct it.